config/defaultMods.json

```diff
@@ -26,7 +26,11 @@
 		"MAX_HEROES_ON_MAP_PER_PLAYER" : 8,
 		"WINNING_HERO_WITH_NO_TROOPS_RETREATS": true,
 		"BLACK_MARKET_MONTHLY_ARTIFACTS_CHANGE": true,
-		"NO_RANDOM_SPECIAL_WEEKS_AND_MONTHS": false
+		"NO_RANDOM_SPECIAL_WEEKS_AND_MONTHS": false,
+		"ATTACK_POINT_DMG_MULTIPLIER": 0.05,
+		"ATTACK_POINTS_DMG_MULTIPLIER_CAP": 4.0,

```

Comment:
what does CAP mean?

Reply:
cap = maximum/limit. "damage cap" is a widely used term for example.